x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/rollup/RollupRestTestStateCleaner.java

```diff
@@ -75,7 +73,7 @@ private void waitForPendingTasks() throws Exception {
     @SuppressWarnings("unchecked")
     private void deleteAllJobs() throws Exception {
         Response response = adminClient.performRequest("GET", "/_xpack/rollup/job/_all");
-        Map<String, Object> jobs = testCase.entityAsMap(response);
+        Map<String, Object> jobs = ESRestTestCase.entityAsMap(response);
```

Comment:
Checkstyle forced me to change this and the rest of the changes to this file come about because of the change here.